### PR TITLE
Create parkour.rs example

### PR DESCRIPTION
**Pull request by emortaldev**

### CODE REVIEW

**Comment by emortaldev** [comment]
I forgot to remove the separate world when the player leaves

**Comment by rj00a** [comment]
> I forgot to remove the separate world when the player leaves

You can still push changes. Don't forget to `cargo fmt`.

**Comment by emortaldev** [comment]
I believe this is ready, worlds are now unloaded when players leave

File: examples/parkour.rs [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+struct ServerState {
+    player_list: Option<PlayerListId>,
+}
+
+#[derive(Clone)]
+struct BlockAndTick {
+    pos: BlockPos,
+    //tick: i64
+}
```
Comment (rj00a) [inline examples/parkour.rs:50]: This could just be a `BlockPos`

**Review by rj00a — CHANGES_REQUESTED** [review]
If you rebase with `main` you'll run into some issues.

Also `cargo clippy --example parkour`.

**Review by rj00a — APPROVED** [review]

**Pull request merged**